### PR TITLE
Adding python-tblib

### PR DESCRIPTION
**Pull request by asmodehn**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1849,6 +1849,9 @@ python-sympy:
     raring: [python-sympy]
     saucy: [python-sympy]
     trusty: [python-sympy]
+python-tblib:
+  ubuntu:
```
Comment (tfoote) [inline rosdep/python.yaml]: This is failing CI due to a trailing space on this line.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -1849,6 +1849,9 @@ python-sympy:
     raring: [python-sympy]
     saucy: [python-sympy]
     trusty: [python-sympy]
+python-tblib:
+  ubuntu: 
+    trusty: [python-tblib]
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: Only adding it for Trusty won't allow you to release any package. Please fill in additional Ubuntu versions.

**Comment by asmodehn** [comment]
Saucy doesn't have this package... I think this might break the deb build on packages that depend on this, is there a way to fix it ?

**Comment by tfoote** [comment]
Generally we want all dependencies to be available on target platforms. 

Typically for something like this we'd consider importing a backport from a ppa if the dependency is necessary and not available otherwise. 

We also plan to turn off Saucy builds since it's reached EOL. Once that has happened builds will work again. But until then we want to keep compatibility.

**Comment by asmodehn** [comment]
So this will be mergeable when saucy is dropped ? any ETA ?
I would like to avoid doing a backport for an obsolete distro...

**Comment by asmodehn** [comment]
Actually, how about I do a third-party release of  https://github.com/ionelmc/python-tblib ?

I recently found out that the trusty version was too old and didn't work as expected... I might need to use a more recent version, but I am not sure how the dependency will be resolved if there is both a rosdep key, and a rosdistro key...

Doing the thirdparty release would also probably satisfy saucy builds...

**Comment by tfoote** [comment]
Yes a 3rdparty build would satisfy saucy and hopefully get you the needed version.

**Comment by asmodehn** [comment]
In the end I might eventually do this a different way...
Anyway I don't need it at the moment.
I ll reopen if needed.

**Pull request closed without merging**